governance/votes/Executive vote - November 26, 2021.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Template - [Executive Vote] Onboarding New Vault Types, Parameter Changes, and Core Unit Budget Distributions - November 26, 2021
+summary: Onboarding wBTC-C and PSM-GUSD-A, linearly increasing the surplus buffer, increasing D3M Debt Ceiling, and distributing Core Unit Budgets.
+date: 2021-11-26T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Onboarding New Vault Types, Parameter Changes, and Core Unit Budget Distributions - November 26, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** and **additions** will occur within the Maker Protocol:
+- The **wBTC-C** vault type will be onboarded to the Maker Protocol with the parameters below.
+- The **GUSD [Peg Stability Module](https://makerdao.world/en/learn/governance/module-psm)** will be onboarded to the Maker Protocol with the parameters below.
+- The **[System Surplus Buffer](https://makerdao.world/en/learn/governance/param-system-surplus-buffer)** will be increased to 90 million DAI over a period of 30 weeks.
+- The **Aave DAI Direct Deposit Module** [Maximum Debt Ceiling](https://makerdao.world/en/learn/governance/module-dciam) will be increased from 50 million DAI to 100 million DAI.
+- **906,805 DAI** total will be transferred to [SAS-001](https://mips.makerdao.com/mips/details/MIP39c2SP25), [IS-001](https://mips.makerdao.com/mips/details/MIP39c2SP24) and [DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23) as per their budget proposals.
```

Comment:
I wonder if "A total of **906,805 DAI** will be transferred..." reads more clearly?

Not much in it tbf

Reply:
Makes sense, changed.